sklearn/linear_model/tests/test_sgd.py

```diff
@@ -476,16 +476,49 @@ def test_n_iter_no_change(klass):
         assert_array_equal(n_iter_list, sorted(n_iter_list))
 
 
-@pytest.mark.parametrize(
-    "klass", [SGDClassifier, SparseSGDClassifier, SGDRegressor, SparseSGDRegressor]
-)
+@pytest.mark.parametrize("klass", [SGDClassifier, SGDRegressor])
```

Comment:
Is there a reason why SparseSGDClassifier and SparseSGDRegressor are removed from this test?